diff --git a/node_modules/vite/dist/node/chunks/node.js b/node_modules/vite/dist/node/chunks/node.js
index 4224827..347f6c1 100644
--- a/node_modules/vite/dist/node/chunks/node.js
+++ b/node_modules/vite/dist/node/chunks/node.js
@@ -34603,7 +34603,6 @@ async function runConfigHook(config, plugins, configEnv) {
 		const res = await getHookHandler(hook).call(context, conf, configEnv);
 		if (res && res !== conf) {
 			if (hasBothRollupOptionsAndRolldownOptions(res)) context.warn(`Both \`rollupOptions\` and \`rolldownOptions\` were specified by ${JSON.stringify(p.name)} plugin. \`rollupOptions\` specified by that plugin will be ignored.`);
-			if (res.esbuild) context.warn(`\`esbuild\` option was specified by ${JSON.stringify(p.name)} plugin. This option is deprecated, please use \`oxc\` instead.`);
 			if (res.optimizeDeps?.esbuildOptions) context.warn(`\`optimizeDeps.esbuildOptions\` option was specified by ${JSON.stringify(p.name)} plugin. This option is deprecated, please use \`optimizeDeps.rolldownOptions\` instead.`);
 			conf = mergeConfig(conf, res);
 		}
